gulpfile.js

```diff
@@ -0,0 +1,37 @@
+var gulp = require('gulp');
+var ts = require('gulp-typescript');
+var sass = require('gulp-sass');
+sass.compiler = require('node-sass');
+var tsProject = ts.createProject('tsconfig.json');
+var paths = {
+    pages: ['src/views/**/*'],
+    public: ['src/public/**/*'],
+    nodeModules: ['node_modules'],
+    govukfrontend: ['node_modules/govuk-frontend/**/*']
+}
+
+gulp.task('build-sass', function () {
+    return gulp.src('./src/public/sass/**/*.scss')
+        .pipe(sass({ includePaths: paths.nodeModules }).on('error', sass.logError))
+        .pipe(gulp.dest('./build/public/css'));
+});
+
+gulp.task('sass:watch', function () {
```

Comment:
A `sass:watch` task does not work due to:

```
./node_modules/.bin/gulp sass:watch
[16:57:42] Using gulpfile ~/Projects/lfp-appeals-frontend/gulpfile.js
[16:57:42] Starting 'sass:watch'...
[16:57:42] 'sass:watch' errored after 1.92 ms
[16:57:42] Error: watching ./src/public/sass/**/*.scss: watch task has to be a function (optionally generated by using gulp.parallel or gulp.series)
    at Gulp.watch (/Users/ddunajski/Projects/lfp-appeals-frontend/node_modules/gulp/index.js:28:11)
    at /Users/ddunajski/Projects/lfp-appeals-frontend/gulpfile.js:20:10
    at sass:watch (/Users/ddunajski/Projects/lfp-appeals-frontend/node_modules/undertaker/lib/set-task.js:13:15)
    at bound (domain.js:429:14)
    at runBound (domain.js:442:12)
    at asyncRunner (/Users/ddunajski/Projects/lfp-appeals-frontend/node_modules/async-done/index.js:55:18)
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
```

I am not sure we need it at this point so might be candidate for removal.

Reply:
Fixed it. I think it's a nice to have as now I know how to do the watch for the `ts` files from gulp